post.css

```diff
@@ -13,7 +15,8 @@ html {
   font-family: var(--rdp-font);
 }
 
-img {
+img,
+video {
```

Comment:
Lets put themes like this into the default css theme so it can be shared accross all pages with media (e.g. the FAQ)